bzt/modules/jmeter.py

```diff
@@ -99,8 +108,12 @@ def prepare(self):
         sys_props = self.settings.get("system-properties")
         props = self.settings.get("properties")
         props_local = scenario.get("properties")
-        if self.distributed_servers and self.settings.get("gui", False):
-            props_local.merge({"remote_hosts": ",".join(self.distributed_servers)})
+        if self.distributed_servers:
+            if self.settings.get("gui", False):
+                props_local.merge({"remote_hosts": ",".join(self.distributed_servers)})
+            else:
```

Comment:
Why under "else"? It is safe to set it always. ANd it must be used always, even for non-distributed tests.